modules/ROOT/pages/do-not-use-to-find-info-with-new-search.adoc

```diff
@@ -0,0 +1,205 @@
+|`/leagues` |Endpoint path
+|League resource |Endpoint display name
+|Information about a league |Endpoint description
+|POST |Endpoint method
+|create team |Endpoint method description
+|offset |Query parameter name, Query parameter description
+|leagueid |URI parameter name
+|team |Type name
+|city |Type property name
+|leagues leagueId teams |Endpoint path
+|`/leagues/LEAGUE_ID/teams` |Endpoint path
+|===
+
+Note that the last two searches are the same, because special characters are removed before searching.
+
+
+== Search by Organizations
+
+You can select which organizations to include in each search. By default, no organization is selected in the menu, and Exchange searches MuleSoft public assets and the assets of the root organization and its business groups.
+
+. Click *Any organization*. 
+. Enter a name of a category or select one or more options from the list.
+. Click *Apply*.
+
+== Search by Categories
+
+You can search for assets by predefined categories. You can search for a combination of categories, and you can search for multiple values for the same category. Multiple values for the same category are combined with the OR operator.
+
+. Click *Any category*. 
+. Enter a name of a category or select one or more options from the list.
+. Click *Apply*.
+
+== Search by Tags
+
+Search by tags that were created by an Exchange administrator or organization owner.
+
+. Click *Any tag*.
+. Enter a name of a tag or select one or more options from the list.
+. Click *Apply*.
+
+
+== Search by Lifecycle States
+
+In the software development lifecycle, assets are developed, released, and eventually deprecated. Each Exchange asset version moves through the lifecycle states of `development`, `stable`, and `deprecated`.
+
+Find asset versions that are ready for consumption, in development, or have been deprecated.
+
+. From the details asset pane, click *Any Lifecycle state*.
+. Select an option from the list and click *Apply*.
+. Assets in that state display in the search results.
+. (Optional) Click *Save this search* to create a shortcut for this search. Saved searches display in the left navigation bar.
+
+
+== Save a Search
+
+To quickly find assets, save shortcuts to filters that you have applied. Saved searches are arranged in alphabetical order in the left navigation bar.
+
+. Enter a search term in the *Search all assets* field and click *Save this search*.
+. Enter the name for the search.
+. Select an option to save as *My search* or an organization search.
+. Click *Save search*.
++
+Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
+. Confirm that your search appears in the saved searches in the left navigation bar.
++
+Exchange lists search terms specific to your business group. Below those terms are saved searches you created:
++
+image::ex2-saved-searches-groups.png[Saved Search Groups]
+
+=== Rename or Delete a Saved Search
```

Comment:
In general, you want to have more than one subtopic before creating subtopics. 
The alternatives are to include a solitary subtopic within its parent, or to create more than one: in this case, you could  have separate rename and delete subtopics?